zulip_bots/zulip_bots/bots/link_shortener/requirements.txt

```diff
@@ -0,0 +1 @@
+request
```

Comment:
This file shouldn't be needed. Aside from the fact that as of now we don't install from each bot's requirements files, the requests module is already a dependency.

You should add it to `zulip_bots/setup.py` instead, for future proofing as follows:


```python
setuptools_info = dict(
    install_requires=[
        'zulip',
        'mock>=2.0.0',
        'html2text',  # for bots/define
    ],
)
```